common/schema-model/src/main/java/io/smallrye/graphql/schema/model/EnumValue.java

```diff
@@ -36,6 +42,20 @@ public void setValue(String value) {
 
     @Override
     public String toString() {
-        return "EnumValue{" + "description=" + description + ", value=" + value + '}';
+        return "EnumValue{" +
+                "description=" + description +
+                ", value=" + value +
+                ", directiveInstances=" +
+                (directiveInstances == null ? "null" : Arrays.toString(directiveInstances.toArray())) +
```

Comment:
```suggestion
                ", directiveInstances=" + directiveInstances +
```

Should look exactly the same: if it's null, it renders "null", if not, it renders the list in square brackets.

Reply:
You're right, I changed it.

Technically it depends on the implementation of List, but everything inheriting from AbstractCollection inherits the square brackets implementation.